src/style.css

```diff
@@ -10,7 +10,6 @@
 
 body {
   font-family: 'Source Sans Pro', sans-serif;
-  overflow-x: hidden;
 }
 
```

Comment:
- [ ] You did a great job styling the page. :+1: Let's work on more things so the design follows the wireframe. 
  - Please move the title `Leaderboard` to the left side. 🤝️
  - The text `Recent Scores` should be aligned to the left relative to the scores below. 🥸️
  - The `Refresh` button should be aligned to the right relative to the scores container. 🙂️

**Your design**
![Screenshot from 2023-05-25 09-06-36](https://github.com/zbahati/leaderboardApi/assets/17243325/cf4d43b6-3d9e-49d3-a966-53495f8b5d2d)

**Wreframe**
![image](https://github.com/zbahati/leaderboardApi/assets/17243325/ed22e591-3337-4c3a-bf2f-92f4c4f0617d)